src/js/components/AccordionPanel/doc.js

```diff
@@ -37,13 +37,23 @@ export const themeDoc = {
     defaultValue: '<FormDown />',
   },
   'accordion.border.color': {
-    description: 'The border color to use in the accordion.',
+    description: 'The panel border color to use in the accordion panel.',
     type: 'string | { dark: string, light: string }',
     defaultValue: 'border',
   },
   'accordion.border.side': {
-    description: 'The border side to use in the accordion.',
+    description: 'The panel border side to use in the accordion panel.',
     type: 'string',
     defaultValue: 'bottom',
   },
+  'accordion.outer.border.color': {
+    description: 'The outer border color around to use in the accordion.',
```

Comment:
Not sure this description makes sense, it will be helpful to understand around when. otherwise not sure what is the difference between the outer and default border. 
If outer is only referring or associated with border it should be nested inside the border, i.e. `border.outer`. But I'm not sure I understand what `outer` means from this description.

Reply:
This border color is referring to the main `box` that holds the whole accordion panel within. The accordion border is referring to the border of the accordion panel so maybe I can change these two descriptions to be border of accordion panel vs border of accordion..